crates/rustc_codegen_spirv/src/linker/structurizer.rs

```diff
@@ -310,27 +522,34 @@ fn block_leads_into_continue(blocks: &[Block], cf_info: &ControlFlowInfo, start:
     false
 }
 
-fn get_possible_merge_positions(
-    blocks: &[Block],
-    cf_info: &ControlFlowInfo,
-    start: Word,
-) -> Vec<usize> {
-    let mut retval = Vec::new();
+// every branch from a reaches b.
+fn block_is_reverse_idom_of(blocks: &[Block], cf_info: &ControlFlowInfo, a: Word, b: Word) -> bool {
```

Comment:
just checking, is this reverse idom or reverse dom? the comment and the function name are contradictory.

Reply:
Yep should be reverse dom. CFG names are hard.